.github/workflows/retest.yaml

```diff
@@ -14,9 +14,10 @@ jobs:
     if: github.repository == 'ceph/ceph-csi'
     runs-on: ubuntu-latest
     steps:
-      # path to the retest action
-      # yamllint disable-line rule:line-length
-      - uses: ceph/ceph-csi/actions/retest@28dc64dcae3cec8d11d84bdf525bda0ef757c688  # devel
+      - name: Checkout the ceph-csi respository
+        uses: actions/checkout@v4
```

Comment:
we should not be using v4 (it should be a hash) here isn't it?

Reply:
indeed, use this:

```yaml
uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
```

Reply:
Ah, right.
Updated it now :)